src/main/java/properties/EnvironmentConfig.java

```diff
@@ -0,0 +1,46 @@
+package properties;
+
+/***
+ * Project environment data
+ */
+public class EnvironmentConfig {
+
+    private static final String MAIN_URL = "url";
+    private static final String REGISTRATION_URL = "regURl";
+    private static final String CATALOGUE_URL = "catalogueURL";
+    private static final String CART_URL = "cartURL";
+    private static final String PASSWORD = "password";
+    private static final String LOGIN = "login";
+
+    private static String FILE_NAME = null;
+
+    private static String environment = System.getProperty("environment");
```

Comment:
Why do you need a global variable for this? You use it just once

Reply:
fixed